sdk/php/template/composer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "daggermodule/example",
+  "description": "An example module you can use as a starting point for building your own dagger modules",
```

Comment:
Doesn't have to be the Module's name btw, it can be something general like "main", but it's been planned for a long time in Python to be able to use the Module's name, so in code you can do `import trivy` instead of `import main`, for example.

If not trivial or if changing the name here has consequences in code, you can choose something static for now like "main", just not "example".

Reply:
It now dynamically generates the package name and has a blank description:

dagger init --sdk=github.com/carnage/dagger/sdk/php@add-php-runtime --name="Potato Pie"

```json
{
  "name": "daggermodule/potato-pie",
  "description": "",
```